src/containers/ReportArticle/state/actions.test.js

```diff
@@ -0,0 +1,103 @@
+import expect from 'expect';
+import MockAdapter from 'axios-mock-adapter';
+import configureMockStore from 'redux-mock-store';
+import thunk from 'redux-thunk';
+import {
+  FETCH_REPORT_TYPES, FETCH_REPORT_TYPES_SUCCESS,
+  REPORTING_FAILURE, REPORTING_SUCCESS,
+} from './types';
+import {
+  reportArticleAction,
+  fetchViolationTypesAction,
+  fetchingViolationTypes,
+  fetchViolationsSuccess,
+  reportArticleSuccess,
+  fetchViolationsFail,
+} from './actions';
+import {
+  reportPayload, errorResp,
+  successResponse, fetchTypesResponse, reportMalPayload,
+} from './mock';
+import api, { getURL } from '../../../utils/api';
+
+const middleware = [thunk];
+const mockStore = configureMockStore(middleware);
+let store;
+
+const callBack = jest.fn();
+
+describe('<ArticleReport /> actions', () => {
```

Comment:
Refactor this test suite title

Reply:
I have refactored it to `The Article Report` as advised. Thanks